SwaprHQ/stackly-ui
# [STK-218][FIX] - Stack orders should show the correct cow order status

## Fixes: [STK-218](https://linear.app/swaprhq/issue/STK-218/ui-should-show-if-it-is-a-cow-order-has-expired)

# Description
* Update the way we handle how to visualize Cow order statuses.

# Visual evidence
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/88678c24-a5ab-4701-85a6-1ec564058de3)

# How to test the changes
**PRE-CONDITIONS:** You need to have existing stacks with executed orders in place to test this fix

1) Pull this branch
2) Run the project locally
3) Go to Stackly dApp
4) Connect your wallet
5) Go to "Your Stacks", choose one of them, and click "View Details"
6) When the Order Details modal opens up, the right-most column should have the correct order status
